packages/webview/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "webview-background",
+  "version": "0.0.2",
+  "description": "A small plugin to change the background of a webview.",
+  "main": "dist/plugin.cjs.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "unpkg": "dist/plugin.js",
+  "files": [
+    "android/src/main/",
+    "android/build.gradle",
+    "dist/",
+    "ios/Sources",
+    "Package.swift",
+    "WebviewBackground.podspec"
+  ],
+  "author": "Matheu DeWitt",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cybersemics/em"
+  },
+  "bugs": {
+    "url": "https://github.com/cybersemics/em/issues"
+  },
+  "keywords": [
+    "capacitor",
+    "plugin",
+    "native"
+  ],
+  "scripts": {
+    "verify": "npm run verify:ios && npm run verify:web",
+    "verify:ios": "xcodebuild -scheme WebviewBackground -destination generic/platform=iOS",
+    "verify:web": "npm run build",
+    "lint": "npm run eslint && npm run prettier -- --check && npm run swiftlint -- lint",
+    "fmt": "npm run eslint -- --fix && npm run prettier -- --write && npm run swiftlint -- --fix --format",
+    "eslint": "eslint . --ext ts",
+    "prettier": "prettier \"**/*.{css,html,ts,js,java}\" --plugin=prettier-plugin-java",
+    "swiftlint": "node-swiftlint",
+    "docgen": "docgen --api WebviewBackgroundPlugin --output-readme README.md --output-json dist/docs.json",
+    "build": "npm run clean && npm run docgen && tsc && rollup -c rollup.config.mjs",
+    "clean": "rimraf ./dist",
+    "watch": "tsc --watch",
+    "prepublishOnly": "npm run build"
```

Comment:
Since we use `yarn` on this project, can we switch to `yarn` instead of `npm`? I think that would be more consistent.